src/Microsoft.PowerShell.ConsoleHost/host/msh/ProgressPane.cs

```diff
@@ -91,32 +91,43 @@ class ProgressPane
 
                 int rows = tempProgressRegion.GetLength(0);
                 int cols = tempProgressRegion.GetLength(1);
-                _location = _rawui.WindowPosition;
-
-                // We have to show the progress pane in the first column, as the screen buffer at any point might contain
-                // a CJK double-cell characters, which makes it impractical to try to find a position where the pane would
-                // not slice a character.  Column 0 is the only place where we know for sure we can place the pane.
-
                 _location.X = 0;
-                _location.Y = Math.Min(_location.Y + 2, _bufSize.Height);
 
 #if UNIX
-                // replace the saved region in the screen buffer with our progress display
-                _location = _rawui.CursorPosition;
+                _location.Y = _rawui.CursorPosition.Y;
 
-                //set the cursor position back to the beginning of the region to overwrite write-progress
-                //if the cursor is at the bottom, back it up to overwrite the previous write progress
-                if (_location.Y >= _rawui.BufferSize.Height - rows)
+                //if the cursor is at the bottom, create screen buffer space by scrolling
+                int scrollRows = rows - ((_rawui.BufferSize.Height - 1) - _location.Y);
+                for (int i = 0; i < rows; i++)
```

Comment:
In rare cases, there may be junk on the screen below the prompt. Maybe we should be writing out spaces to make sure it's clear.